atlasdb-ete-tests/docker-compose.multiple-cassandra.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+
+services:
+  cassandra1:
+     image: com.palantir.docker/atlas-cassandra:2.2-v0.1
+     environment:
+      - CASSANDRA_SEEDS=cassandra1,cassandra2
```

Comment:
Should we include cassandra3 as a seed or are we testing the cluster gossiping correctly to pick it up?